### ISSUE TITLE
PI with random start values - enable selection of parameters which should be randomly drawn

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/MoBi#138]
It would be very useful to select which parameter should be random and which should start with the defined values.

**Comment by Yuri05** [comment]
Isn't this the case when we set some of the identification parameters to "Fixed"??

**Comment by PavelBal** [comment]
No. When a parameter is "fixed", it will not be varied at all. This feature is about having some parameters randomly drawn in the "multiple start values optimiziation" mode, while the marked parameters always start with the defined value.

**Comment by Yuri05** [comment]
>When a parameter is "fixed", it will not be varied at all.  

It will be set to the initial value defined in the PI, which is exactly
>while the marked parameters always start with the defined value.

Or do I miss something?

**Comment by PavelBal** [comment]
The search space of the marked parameter shoul start with the defined value, but the parameter should still be optimized.
